packages/ui-components/src/components/search.tsx

```diff
@@ -215,6 +215,26 @@ export const FilterBox = (props: IFilterBoxProps): JSX.Element => {
 /**
  * A widget which hosts a input textbox to filter on file names.
  */
-export const FilenameSearcher = (props: IFilterBoxProps): ReactWidget => {
-  return ReactWidget.create(<FilterBox {...props} />);
-};
+export class FilenameSearcher extends ReactWidget {
+  private filterBoxProps: IFilterBoxProps;
```

Comment:
Let's follow convention of prefixing private properties with `_` and placing them at the bottom

```suggestion
  private _filterBoxProps: IFilterBoxProps;
```